src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -101,7 +101,15 @@ private void replaceText(String text) {
     @FXML
     private void handleCommandEntered() {
         try {
-            CommandResult commandResult = logic.execute(commandTextField.getText());
+            String command = commandTextField.getText();
+            CommandResult commandResult;
+            String[] commandWord = command.trim().split(" ");
```

Comment:
This solution is probably fine for now, since the main priority is for the jarfile to work on Thursday. But we would need to refactor this into a more 'correct' solution (like adding some functionality into Logic to check the type of command before passing it to the respective model, or making the 2 models inherit from the same interface or something else).

Reply:
I think these lines are checking the style of command. can be move into logic. I don't recommend  change the model since model is used in all parts, if changing the model, the whole thing will be changed.